src/mainboard/clevo/adl-p/variants/nv40pz/hda_verb.c

```diff
@@ -6,7 +6,11 @@ const u32 cim_verb_data[] = {
 	/* Realtek, ALC256 */
 	0x10ec0256, /* Vendor ID */
 	0x15584041, /* Subsystem ID */
-	12, /* Number of entries */
+	18, /* Number of entries */
+	0x0205001a,
+	0x02048003,
+	0x0205001a,
+	0x0204c003,
```

Comment:
It would be nice if we could use a macro for these

Reply:
These are vendor specific verbs not standard azalia verbs, you would have to find datasheet to know what that means.